.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: Go
```

Comment:
Please update this file type to `.yaml`

Reply:
this file type is .yaml. This is the name of the ... action .... won't it be weird to have an action called go.yaml?

Reply:
wait ... this type is .... yml ......